app/views/funding/payment_schedules/_month_breakdown.html.erb

```diff
@@ -22,7 +22,7 @@
           <tr class="govuk-table__row">
             <th scope="col" class="govuk-table__header app-table__column-50">Payment type</th>
             <th scope="col" class="govuk-table__header govuk-table__header--numeric app-table__column-25">Amount</th>
-            <th scope="col" class="govuk-table__header govuk-table__header--numeric app-table__column-25">Running total</th>
+            <th scope="col" class="govuk-table__header govuk-table__header--numeric app-table__column-25">Running total for payment type</th>
```

Comment:
I wonder if we could reduce the letter count here somehow, to stop the heading from wrapping to the next line? Maybe we can just say 'Running total for type' but not sure what Maeve thinks?

<img width="701" alt="Screenshot 2022-06-06 at 11 29 34" src="https://user-images.githubusercontent.com/43522239/172144409-6ad15c44-bb79-445e-9878-100c3a684826.png">

Reply:
Maeve and I did have a chat about this! cc @MaeveRoseveare

Reply:
Alternatively we could set a no wrap class on `payment type` so at least it stays together.

Reply:
I think saying 'payment type' is more clear than just 'type' even though it's longer. A no wrap class could work like Ed suggested, I think it would read better too.

Reply:
Added no-wrap